app/components/zitting-text-document-container.js

```diff
@@ -0,0 +1,58 @@
+import Component from '@glimmer/component';
+import { task } from "ember-concurrency-decorators";
+import { action } from "@ember/object";
+
+export default class ZittingTextDocumentContainerComponent extends Component {
+  constructor(...args) {
+    super(...args);
+    this.getText.perform();
+  }
+
+  profile = 'none';
+  editor;
+  type = this.args.type;
+  text;
+
+  editorOptions = {
+    showToggleRdfaAnnotations: false,
+    showInsertButton: true,
+    showRdfa: false,
+    showRdfaHighlight: false,
+    showRdfaHover: false
+  };
+
+  @task
+  *getText() {
+    const zitting = yield this.args.zitting;    
+    if(zitting && !zitting.intro){
```

Comment:
Instead of catching the case where a zitting record doesn't have a certain attribute value, I would add empty strings for these on record creation. An alternative could be to give them default values in the Model class definition.